packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -222,26 +224,25 @@ export class FluidClientDebugger
 	 *
 	 */
 	private readonly postAudienceStateChange = (): void => {
-		console.log("---------------------------------------");
-		console.log("this.container.audience.getMembers():", this.container.audience.getMembers());
-		console.log("---------------------------------------");
-		console.log("this.getAudienceHistory():", this.getAudienceHistory());
-		console.log("---------------------------------------");
+		const audienceMembersMap = this.container.audience.getMembers();
+		const audienceMembersArray: IClient[] = [...audienceMembersMap.values()];
```

Comment:
General comment on this, not specific to this PR... what would we expect in terms of audience size, in production, for very big/extremely-shared documents? Hundreds of clients seems reachable. Probably not thousands, unless *a lot* of people leave browser tabs open on the same shared doc and the app does not do idle disconnection, or something?

That said, the debugger is probably more targeted to local dev scenarios where audience should be pretty small most of the time.

Just something to think about when we're passing messages that could get quite big. Probably also applies to a container's summary.

cc @Josmithr @rohandubal

Reply:
Yeah, my assumption is that we would start with the naive approach (send complete summaries on each change), and consider an incremental approach if/when needed. But I suspect we will eventually want to address this.